SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -292,10 +292,7 @@ public async Task<IActionResult> ConfirmAddress_Post(ConfirmAddressViewModel vie
             {
                 propertyData.Epc = await epcApi.GetEpcForId(viewModel.SelectedEpcId);
             }
-            else
```

Comment:
Why remove this?

Reply:
it's redundant given back buttons only take users back to FindEPC (though this is pending that discussion of course). At the moment epc is cleared on FindEpc_Post so the user can't get to the postcode page with a non-null epc